### PR TITLE
runtime: Fix firecracker config

### PR DESCRIPTION
**Pull request by jodh-intel**
The build was setting a `FCVALIDPATHS` variable for firecracker, but
that was never being used. Conversely, the firecracker configuration
template was expecting a `FCVALIDHYPERVISORPATHS`, but that variable was
never being set.

Resolve by only setting the `FCVALIDHYPERVISORPATHS` variable to ensure
the generated firecracker config is valid once again.

Fixes: #1001.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @jodh-intel!

**Comment by jodh-intel** [comment]
These fixes should get applied to https://github.com/kata-containers/runtime/pull/3005 so this PR won't need to be backported to 1.x. However, it will need to be backported to the stable 2.0 branch in this repo.

I don't know who created the [`stable-2.0.0` branch](https://github.com/kata-containers/kata-containers/tree/stable-2.0.0) (?), but it needs renaming to `stable-2.0` to match all the other stable branches imho.

@bergwolf - do you know anything about this?

**Review by c3d — APPROVED** [review]
Ooops, my bad. Good catch.

**Review by c3d — APPROVED** [review]

File: src/runtime/Makefile [hunk 1]
```diff
@@ -139,10 +139,10 @@ QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
 QEMUVALIDVIRTIOFSPATHS := $(QEMUBINDIR)/$(QEMUVIRTIOFSCMD)
 
 CLHPATH := $(CLHBINDIR)/$(CLHCMD)
-CLHVALIDHYPERVISORPATHS := [\"$(CLHBINDIR)/$(CLHCMD)\"]
+CLHVALIDHYPERVISORPATHS := [\"$(CLHPATH)\"]
 
 FCPATH = $(FCBINDIR)/$(FCCMD)
-FCVALIDPATHS = [\"$(FCPATH)\"]
+FCVALIDHYPERVISORPATHS := [\"$(FCPATH)\"]
```
[the remark this answers is not in the document]
Comment (jodh-intel) [reply src/runtime/Makefile:145]: @c3d - For consistency with the other `*VALIDHYPERVISORPATHS` variables:

```bash
$ grep "VALIDHYPERVISORPATHS.*=" src/runtime/Makefile
QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
CLHVALIDHYPERVISORPATHS := [\"$(CLHPATH)\"]
FCVALIDHYPERVISORPATHS := [\"$(FCPATH)\"]
ACRNVALIDHYPERVISORPATHS := [\"$(ACRNPATH)\"]
```

**Review by devimc — APPROVED** [review]

**Comment by devimc** [comment]
restarting clh CIs

File: src/runtime/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (c3d) [reply src/runtime/Makefile:145]: OK. Will not repair the global inconsistency just yet. Maybe later.

**Comment by amshinde** [comment]
@jodh-intel Needs port to stable 2.0.0 branch.

**Pull request merged**